superbench/runner/runner.py

```diff
@@ -210,6 +211,17 @@ def fetch_results(self):    # pragma: no cover
             )
         )
 
+    def generate_ibstat(self):    # pragma: no cover
+        """Generate ibstat file and distruibute to all nodes."""
+        logger.info('Generating ibstat file to all nodes.')
+        self._ansible_client.run(
+            self._ansible_client.get_playbook_config(
+                'generate_ibstat.yaml', extravars={
+                    'workspace': str(os.environ['HOME']),
```

Comment:
We should keep all file under SuperBench folder

Reply:
Has been changed to SuperBench folder, thanks.